packages/ibm-products/src/components/InlineTip/InlineTip.mdx

```diff
@@ -0,0 +1,53 @@
+import { Story, ArgsTable, Source, Canvas } from '@storybook/addon-docs';
```

Comment:
With the current Storybook 7 configuration, these docs will need to be moved to `InlineTip.docs-page.js` and you can delete this file.